articles/extensions/custom-social-extensions.md

```diff
@@ -92,3 +102,19 @@ If the identity provider requires Basic Authentication, you can be use the __Cus
     "Authorization" : "Basic MTIzNDU2OmFiY2RlZg=="
 }
 ```
+
+## Additional Provider Resources

```

Comment:
Not sure about this paragraph. It can be helpful to users for sure but it looks like content they can easily find, and if we leave it here we will have to maintain it. For example, when you update the image with the latest, you will have to add links here as well.